src/Api.js

```diff
@@ -24,7 +24,7 @@ export const getSingleArticleComments = (article_id) => {
 
 export const patchArticleVote = (article_id) => {
     const patchRequestedArticleVote = {
-        inc_votes: 1
+        inc_votes: 'hello'
```

Comment:
why is this now 'hello' - now the request will always fail as it's a bad request this should always be a number but we want to catch it if it fails for some other reason, i.e a user being offline